docs/release-notes/release-2024-07-16.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 0.58 - July 2024
+---
+
+:::warning
+This release contains breaking changes
+:::
+
+## Examples in TypeSpec
+
+This release introduce a new `@example` and `@opExample` decorator built using the new values introduced in last release(0.57.0).
```

Comment:
```suggestion
This release introduce a new `@example` and `@opExample` decorator.
```

Just feel like this is potentially confusing ("value" is not a particularly descriptive word)